api/python/quilt3/workflows/config-1.schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://quiltdata.com/workflows/config/1",
+  "type": "object",
+  "required": [
+    "version",
+    "workflows"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "version": {
+      "const": "1"
+    },
+    "is_workflow_required": {
+      "type": "boolean",
+      "description": "Whether package can be pushed with unspecified workflow.",
+      "default": false
+    },
+    "default_workflow": {
+      "type": "string",
+      "description": "The workflow that will be used if no workflow provided by user."
+    },
+    "workflows": {
+      "type": "object",
+      "minProperties": 1,
+      "propertyNames": {
+        "pattern": "^[A-Za-z_-][A-Za-z0-9_-]*$",
+        "maxLength": 64
+      },
+      "additionalProperties": {
+        "type": "object",
+        "required": [
+          "name"
+        ],
+        "properties": {
+          "name": {
+            "type": "string",
+            "description": "The workflow name for UI.",
+            "minLength": 1
+          },
+          "description": {
+            "type": "string",
+            "description": "The workflow description for UI.",
+            "minLength": 1
+          },
+          "metadata_schema": {
+            "type": "string",
+            "description": "JSON Schema $id."
+          },
+          "is_message_required": {
+            "type": "boolean",
+            "description": "Whether commit message is required.",
+            "default": false
+          }
+        }
+      }
+    },
+    "schemas": {
+      "type": "object",
+      "description": "JSON Schemas to be used for metadata validation.",
+      "minProperties": 1,
+      "additionalProperties": {
+        "type": "object",
+        "required": [
+          "url"
+        ],
+        "properties": {
+          "url": {
+            "type": "string",
+            "description": "URL from where the schema will be obtained.",
```

Comment:
> Is this in fact a URL?

https://tools.ietf.org/html/rfc3986#section-1.1.3
>    A URI can be further classified as a locator, a name, or both.  The
   term "Uniform Resource Locator" (URL) refers to the subset of URIs
   that, in addition to identifying a resource, provide a means of
   locating the resource by describing its primary access mechanism
   (e.g., its network "location").

> I thought we were requiring the schemas to be in S3?

For local repositories we also support `file:///some/path`. 

> If so we should have a more precise format?

I'm not sure, currently I feel it like more implementation detail rather than design choice.